src/org/zaproxy/zap/extension/api/CoreAPI.java

```diff
@@ -953,7 +953,7 @@ private void getURLs(SiteNode parent, ApiResponseList list) {
 	private ApiResponseSet alertToSet(Alert alert) {
 		Map<String, String> map = new HashMap<>();
 		map.put("id", String.valueOf(alert.getAlertId()));
-		map.put("alert", alert.getAlert());
```

Comment:
The "alert" property should still be returned, to keep compatibility with older tools.